### DISCUSSION TITLE
v1.2.1

### DISCUSSION THREAD

**Discussion by gabbybilka** [microsoft/WindowsAppSDK#3156] [Announcements]
## Windows App SDK 1.2.1 (1.2.221116.1) is now available

Windows App SDK 1.2.1 is a stable servicing release that fixes a critical bug for 1.2:

Fixed issue that caused a crash on startup in C++ WinUI 3 apps when adding a WebView2 or TextBox control. For more information see GitHub issues [#7911](https://github.com/microsoft/microsoft-ui-xaml/issues/7911) & [#3117](https://github.com/microsoft/WindowsAppSDK/issues/3117).

📝 [Release notes](https://aka.ms/windowsappsdk/1.2/1.2.221116.1/release-notes)
📦 [1.2.221116.1 NuGet](https://www.nuget.org/packages/Microsoft.WindowsAppSDK/1.2.221116.1)

- If you're building an app with the Windows App SDK for the first time, follow the [installation instructions here](https://docs.microsoft.com/windows/apps/winui/winui3/create-your-first-winui3-app?tabs=desktop-csharp).

- If you are already using Windows App SDK 1.2 and would like to update to the latest release, you can do so by following the [update instructions here](https://docs.microsoft.com/windows/apps/windows-app-sdk/update-existing-projects-to-the-latest-release).

<hr /><em>This discussion was created from the release <a href='https://github.com/microsoft/WindowsAppSDK/releases/tag/'>v1.2.1</a>.</em>

**Comment by Marv51** [comment]
Does somebody know, why the MSIX for this release has the version: "2000.684.1510.0" I thought the 1.2.1 release would be 2001.x.y.0?
